server/constants/collectives.js

```diff
@@ -4,9 +4,12 @@ export const types = {
   COLLECTIVE: 'COLLECTIVE',
   EVENT: 'EVENT',
   USER: 'USER',
-  ORGANIZATION: 'ORGANIZATION'
+  ORGANIZATION: 'ORGANIZATION',
+  BOT: 'BOT'
```

Comment:
I think this is a good idea and will be reused in the future.

Reply:
👍

Reply:
Just need to watch out for potential issues in the frontend code that may expect `ORGANIZATION` or `USER` (a quick check feels like it should be ok but we need to test).

I also like this pattern to create 3rd party bots or apps. One could add such a bot as a member of the collective or host organization to activate the bot. And the bot could perform tasks that any other member/admin (depending on the permissions given to the bot when added as a member of a collective) could do.